### DISCUSSION TITLE
Is there a tutorial on how to use the cheats options on savegame?

### DISCUSSION THREAD

**Discussion by FrankAzafrank** [bucanero/apollo-ps3#127] [Q&A] [answered]
I tried to apply cheats on different games with this app, but after selecting "save and resign" it just stays loading "applying changes" but nothing happens.

**Comment by bucanero** [comment]
the cheat code could be invalid, the file might not be compatible with the cheat, or it could be a bug
But you'll have to provide more information or details, like which game, code, and save are you trying to use.

You should get the debug logs from the app and share the .log file:
- go to "Settings" menu
- select "enable debug log"
- then go to the save you want to modify, select cheats, and apply 

then if the error happens, go and upload the .log file here on Github so it can be reviewed.

You can also ask on the official Discord channel: https://discord.gg/save-wizard `#apollo-save-tool`

**Reply by FrankAzafrank** [reply]
Thank you for the fast reply! I'll try with the same games and if the problem persists I'll share the log and everything else you asked.

**Comment by FrankAzafrank** [comment]
[apollo.log](https://github.com/bucanero/apollo-ps3/files/12775374/apollo.log)
I tried using the cheats again in Fight Night Champion but to no avail, the problem persisted. I'm new to all of this encryption and save file management. I attached the log file created by apollo today

**Reply by bucanero** [reply]
the log is incomplete, you didn't open the save list or applied the cheats after enabling debug log.

One more time:
- start Apollo from XMB
- go to "Settings" menu
- select "enable debug log"
- go to the "HDD Saves", check the list of saves
- then go to the save you want to modify, select cheats, and apply

Without proper details I can't analyze it. (the log should include the save TITLE-ID, the cheats you tried to apply, and more debugging information)

**Reply by FrankAzafrank** [reply]
I'm terribly sorry... Here's the updated log file
[apollo.log](https://github.com/bucanero/apollo-ps3/files/12777443/apollo.log)
Again I'm sorry for bothering and thank you for being patient with me.

**Reply by bucanero** [reply]
the file is still the same, I can't do anything with that file. Go to Discord and ask the community, I can't help.

**Reply by FrankAzafrank** [reply]
Can you provide a discord invite link? Couldn't find it anywhere and this link https://discord.com/channels/347662863986327555/1097201528466587678 you posted above takes me nowhere, says "No text channels"

**Reply by bucanero** [reply]
https://discord.gg/save-wizard , channel `#apollo-save-tool`

**Reply by bucanero** [reply, marked as answer]
For future reference, the bug was due to an invalid formatting character on the `BLUS30608.savepatch`.

The save patch files have been fixed and you can auto-update Apollo Save Tool over the internet:
- open Apollo
- go to "Settings"
- select "Update application data"
- wait for download & install 

Once the update pack is installed, applying changes to Fight Night Champion should work as expected.

**Comment by Simon-ghoster** [comment]
Hi , is there a way that i can edit or add cheats in apollo save tool ?!

Regards.

**Reply by bucanero** [reply]
> Hi , is there a way that i can edit or add cheats in apollo save tool ?!
> 
> Regards.

All the cheat patches can be found here: https://github.com/bucanero/apollo-patches

You can create your own files too, and then just copy the files the Apollo folder on your PS3
